Datadog.Trace.Security.sln

```diff
@@ -37,8 +37,7 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "test-applications", "test-applications", "{9518425A-36A5-4B8F-B0B8-6137DB88441D}"
 	ProjectSection(SolutionItems) = preProject
 		test\test-applications\Directory.Build.props = test\test-applications\Directory.Build.props
-	EndProjectSection
-EndProject
+	EndProjectSectionEndProject
```

Comment:
This looks like a typo. `EndProjectSectionEndProject` => `EndProjectSection` and `EndProject`

Reply:
Thanks, correct.